ssn/rdf/ontology/core/sosa-observation.ttl

```diff
@@ -30,7 +30,7 @@ sosa:Observation
   a owl:Class ;
   rdfs:label "Observation"@en ;
   skos:definition """
-  Act of carrying out an (Observation) Procedure to estimate or calculate a value of a property of a FeatureOfInterest. Links to a Sensor to describe what made the Observation and how; links to a Property to describe what the result is an estimate of, and to a FeatureOfInterest to detail what entity that property was associated with.
+  An act of carrying out an observing Procedure using a Sensor to estimate or calculate a value of an observable Property of a FeatureOfInterest.  
   """@en ;
   skos:note """
   When the feature of interest cannot be observed directly, a sample of it might be used in Observations as a proxy for it. In such a case the Sample is the proximate Feature of Interest of the Observation, while the (ultimate) Feature of Interest of the act of Sampling (i.e. the entity that the Sample is ultimately 'isSampleOf') is the ultimate Feature of Interest of the Observation.
```

Comment:
```suggestion
  When the feature of interest cannot be observed directly, a sample of it might be used in Observations as a proxy for it. In such case, the Sample is the proximate Feature of Interest of the Observation, while the (ultimate) Feature of Interest of the act of Sampling (i.e., the entity that the Sample is ultimately 'isSampleOf') is the ultimate Feature of Interest of the Observation.
```